src/main/java/domain/Line.java

```diff
@@ -0,0 +1,48 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Line {
+
+    private final List<Bridge> bridges;
+
+    private Line(List<Bridge> bridges) {
+        this.bridges = bridges;
+    }
+
+    public static Line createByStrategy(final BridgeGenerator bridgeGenerator, final int personCount) {
+        List<Bridge> bridges = new ArrayList<>();
+        while (bridges.size() != personCount - 1) {
```

Comment:
```suggestion
        while (bridges.size() < personCount - 1) {
```
while문 반복 조건은 이렇게 수정되는 게 맞겠네요.

Reply:
리스트의 사이즈가 0에서부터 시작하니 같은 코드이지 않을까 생각하고 코딩했었는데, 혹시나 리스트의 사이즈가 경계값을 넘길 수 있는 위험이 추후에 발생할 가능성이 0이 아니므로 무한루프를 피하기 위해 반복 조건은 제시해주신 것처럼 `작을 동안`으로 명확해야겠네요!